cmd/juju/model/defaults.go

```diff
@@ -37,24 +38,24 @@ supplied key to the supplied value. This can be repeated for multiple keys.
 You can also specify a yaml file containing key values.
 By default, the model is the current model.
 
-Model default configuration settings are specific to the cloud on which the model runs.
-If the controller host more then one cloud, the cloud (and optionally region) must be specified.
+Model default configuration settings are specific to the cloud on which the
+model runs.
```

Comment:
Wondering if `s/model runs/model is deployed` or something similar would be better.